### PR TITLE
Feat: Fix default diam distr

### PR DESCRIPTION
**Pull request by arnaudon**
Fixing the mistakes of PR #47

### CODE REVIEW

File: neurots/schemas/distributions.json [hunk 1]
```diff
@@ -206,16 +206,36 @@
             "description": "The diameter specifications",
             "oneOf": [
                 {
-                    "additionalProperties": true,
+                    "additionalproperties": true,
```
Comment (adrien-berchet) [inline neurots/schemas/distributions.json:209]: The upper case is needed here.

File: neurots/schemas/distributions.json [hunk 2]
```diff
@@ -206,16 +206,36 @@
...
                             "enum": [
-                                "external", "default"
+                                "external"
                             ],
                             "type": "string"
                         }
                     }
                 },
+                {
+                    "additionalproperties": false,
```
Comment (adrien-berchet) [inline neurots/schemas/distributions.json:220]: The upper case is needed here.

File: neurots/schemas/distributions.json [hunk 3]
```diff
@@ -206,16 +206,36 @@
...
+                    "properties": {
+                        "method": {
+                            "enum": [
+                                "default"
+                            ],
+                            "type": "string"
+                        },
+                        "apical_dendrite": {
+                            "type": "array"
```
Comment (adrien-berchet) [inline neurots/schemas/distributions.json:229]: You should add a `items` entry to describe the schema for the list elements

**Comment by arnaudon** [comment]
@adrien-berchet , good to merge and release?

**Review by adrien-berchet — APPROVED** [review]
LGTM!
Did you test it in region-grower and synthesis-workflow?

**Comment by arnaudon** [comment]
Cool, I am fixing them atm and checking!

**Pull request merged**